decidim-core/app/views/decidim/account/_account_notification.js.erb

```diff
@@ -0,0 +1,20 @@
+var hideExistingNotification = document.querySelector("main").children[0]
```

Comment:
Could you instead do something like `document.querySelector("main > .flash.callout")`?

This way you could avoid the below class checks.